irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,236 @@
+$('#btn-signin-irods').on('click',function(){
+    if(sessionStorage.username)
+        $("#username").val(sessionStorage.username);
+    if(sessionStorage.password)
+        $("#password").val(sessionStorage.password);
+    if(sessionStorage.host)
+        $("#host").val(sessionStorage.host);
+    if(sessionStorage.port)
+        $("#port").val(sessionStorage.port);
+    if(sessionStorage.zone)
+        $("#zone").val(sessionStorage.zone);
+});
+
+$('#btn-select-irods-file').on('click',function(){
+    $('#res_type').val($('#resource-type').val());
+    $('#file_struct').children().remove();
+    if (sessionStorage.signininfo) {
+        $("#irods_content_label").text(sessionStorage.username);
+        $('#root_store').val(sessionStorage.datastore);
+    }
+	store = $('#root_store').val();
+	// Setting up the view tab
+	$('#file_struct').attr('name',store);
+	$('#irods_view_store').val(store);
+	// loading file structs
+	parent = $('#file_struct');
+	get_store(store,parent,0);
+	enable_settings();
+});
+
+function get_store(store, parent, margin){
+	form_data = new FormData();
+	form_data.append('store', store);
+    form_data.append('user', sessionStorage.username);
+    form_data.append('password', sessionStorage.password);
+    form_data.append('zone', sessionStorage.zone);
+    form_data.append('port', sessionStorage.port);
+    form_data.append('host', sessionStorage.host);
```

Comment:
FormData is a new API and not supported in versions of Internet Explorer before version 10. I don't know what Hydroshare's minimum supported browsers are (this should be clearly documented if it is now) but I think supporting only 10 and up is a very reasonable policy, I just want to be sure this is considered.